modules/utils/CMakeLists.txt

```
@@ -47,7 +47,7 @@ set(SOURCES
 # library target
 define_module_library(
   NAME utils
-  PUBLIC_LINK_LIBS absl::random_random absl::time fmt::fmt
+  PUBLIC_LINK_LIBS absl::random_random absl::time fmt::fmt absl::log
```

Could you please add also, I forgot it from the bit flag PR
```suggestion
  PUBLIC_LINK_LIBS absl::random_random absl::time fmt::fmt absl::log magic_enum::magic_enum
```